edweiss-app/components/courses/AssignmentDisplay.tsx

```diff
@@ -98,6 +111,7 @@ const AssignmentDisplay: ReactComponent<{ item: AssignmentWithColor, id: string,
                     swipeableRefs.current[index] = ref;
                 }}
                 renderRightActions={renderRightActions}
+                renderLeftActions={isTeacher ? renderLeftActions : undefined}
```

Comment:
You made this only possible as being a professor. That's a great idea. That would have been fun if a student can `edit` it  😋.